windows/src/developer/TIKE/project/Keyman.Developer.UI.Project.UfrmProject.pas

```diff
@@ -460,6 +460,20 @@ procedure TfrmProject.WebCommandProject(Command: WideString; Params: TStringList
     else if SelectedMRUFileName <> '' then
       OpenContainingFolder(SelectedMRUFileName);
   end
+  else if Command = 'openbuildfolder' then
+  begin
+    pf := SelectedProjectFile;
+    if Assigned(pf) and (pf is TkmnProjectFile) then
+      OpenContainingFolder((pf as TkmnProjectFile).TargetFileName)
+    else if Assigned(pf) and (pf is TkpsProjectFile) then
+      OpenContainingFolder((pf as TkpsProjectFile).TargetFileName)
+    else if Assigned(pf) and (pf is TmodelTsProjectFile) then
+      OpenContainingFolder((pf as TmodelTsProjectFile).TargetFileName);
```

Comment:
While I could have refactored `TargetFileName` into a property on the parent class, I didn't want to make that change for the sake of these 6 lines of code at this point -- there's just too much risk for that kind of refactor at this point.